Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContext.mm

```diff
@@ -60,14 +60,14 @@ @implementation _WKWebExtensionContext
 
 #if ENABLE(WK_WEB_EXTENSIONS)
 
-+ (instancetype)contextWithExtension:(_WKWebExtension *)extension
++ (instancetype)contextForExtension:(_WKWebExtension *)extension
```

Comment:
It's a bit strange, I think `contextForExtension` sounds better than `contextWithExtension`, but I think `initWithExtension` sounds better than `initForExtension`.

I think they should be consistent though, and I think `For` is more clear across both.

Reply:
Agreed.